platform/microprofile/src/test/liberty/config/server.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<server description="Trellis Server">
+    <featureManager>
+      <feature>cdi-2.0</feature>
```

Comment:
Is there a reason you are using individual feature selections (as opposed to `microProfile-n.n`)?

Reply:
That's part of the WIP-ness of this branch. Here, I am using individual feature selections just to avoid the OpenAPI errors that are generated when using the full microProfile feature. Eventually, yes, we would want to just reference `microProfile-n.m`.

Reply:
Cool.